FlightOps/book/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class BookConfig(AppConfig):
```

Comment:
```suggestion
from django.apps import AppConfig

class BookConfig(AppConfig):
```

Espaço desnecessario